api/proto/tournament_service/tournament_service.proto

```diff
@@ -52,6 +60,10 @@ message TournamentPairingRequest {
   int32 round = 5;
 }
 
+// TournamentResultOverrideRequest is sent when a score needs to be edited
+// or added by a director. Note that player one and player two need to be
+// the players that went first and second, respectively; otherwise,
+// we will not be able to accurately track firsts/seconds.
```

Comment:
Order does not matter for submitting results. Order only matters when a director overrides or sets a pairing with the SetPairing service.